mackup/applications/starship.cfg

```
@@ -0,0 +1,5 @@
+[application]
+name = Starship
+
+[configuration_files]
+.config/starship.toml
```

I believe this should use the [built-in XDG support](https://github.com/lra/mackup/tree/master/doc#add-support-for-an-application-using-the-xdg-directory) instead: 
```suggestion
[xdg_configuration_files]
starship.toml
```

---

seems starship is not using XDG: https://github.com/starship/starship/issues/896

---

Oh, indeed. I just assumed it was since the file is in `.config`. Sorry about that!

---

Yeah it's super confusing.